### PR TITLE
DOC-724: Convert JobTask to markdown

### PR DESCRIPTION
**Pull request by taylorhixson**
Update the docs: 
- Convert `jobtask-reference.md` to Markdown and review the functions.
- Add `map_results`, `plot_quicklooks`, `plot_results` functions to `jobtask-reference.md`
- Edit `structure.md` to update JobTask class overview
- Edit `job-reference.md` to add link to and information about JobClass information.

Corresponds to Documentation JIRA task [DOC-724](https://up42.atlassian.net/browse/DOC-724).

[DOC-724]: https://up42.atlassian.net/browse/DOC-724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/structure.md [hunk 1]
```diff
@@ -191,10 +191,27 @@
 
     === "JobTask"
 
-        {{ docstring_jobtask }}
-        <br>
-        Available functions, see also [**JobTask reference**](jobtask-reference.md):
-        {{ format_funcs(funcs_jobtask) }}
+        The JobTask class enables access to [results of a specific job task](../../analytics/).
```
Comment (dustgalactic) [inline docs/structure.md:194]: The link doesn't work.

File: docs/structure.md [hunk 2]
```diff
@@ -191,10 +191,27 @@
...
+
+        Job tasks are unique configurations of workflow tasks in a job.
+
+        ```python
+        jobtask = up42.initialize_jobtask(
+            jobtask_id="3f772637-09aa-4164-bded-692fcd746d20",
+            job_id="de5806aa-5ef1-4dc9-ab1d-06d7ec1a5021"
```
Comment (dustgalactic) [inline docs/structure.md:201]: The second argument needs a comma at the end as well

File: docs/structure.md [hunk 3]
```diff
@@ -191,10 +191,27 @@
...
+        )
+        ```
+
+        See available attributes and functions on the [JobTask](jobtask-reference.md) reference page:
+        <ul>
+            <li>`info`</li>
+            <li>`download_quicklooks()`</li>
+            <li>`get_results_json()`</li>
+            <li>`download_results()`</li>
+            <li>`map_results()`</li>
```
Comment (dustgalactic) [inline docs/structure.md:211]: It should be instead
plot_quicklooks()
map_results()
plot_results()

File: docs/reference/jobtask-reference.md [hunk 4]
```diff
@@ -1,6 +1,179 @@
...
-::: up42.jobtask.JobTask
-    selection:
-        inherited_members: True
-        members: {{ funcs_jobtask }}
+The JobTask class enables access to [results of a specific job task](analytics.md). Job tasks are unique configurations of workflow tasks in a job.
+
+```python
+jobtask = up42.initialize_jobtask(
+    jobtask_id="3f772637-09aa-4164-bded-692fcd746d20",
+    job_id="de5806aa-5ef1-4dc9-ab1d-06d7ec1a5021"
```
Comment (dustgalactic) [inline docs/reference/jobtask-reference.md:8]: The second argument needs a comma at the end

File: docs/reference/jobtask-reference.md [hunk 5]
```diff
@@ -1,6 +1,179 @@
...
+)
+```
+
+## Job tasks
+
+### info
+
+The `info` attribute returns metadata to a specific job task.
```
Comment (dustgalactic) [inline docs/reference/jobtask-reference.md:16]: of a specific job task

File: docs/reference/jobtask-reference.md [hunk 6]
```diff
@@ -1,6 +1,179 @@
...
+
+<h5> Example </h5>
+
+```python
+jobtask.info
+```
+
+### download_quicklooks()
+
+The `download_quicklooks()` function allows you to download low-resolution preview images. Not all job tasks have quicklooks available.
```
Comment (dustgalactic) [inline docs/reference/jobtask-reference.md:28]: Not all job tasks have quicklooks.

File: docs/structure.md (hunk 1 above)
Comment (taylorhixson) [reply docs/structure.md:194]: Got it! Working locally for me now. Marking resolved, but please let me know if there are still problems on the next review :)

**Review by dustgalactic — APPROVED** [review]

**Review by seedlit — APPROVED** [review]
looked over the surface.

**Pull request merged**